src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -78,6 +82,24 @@ public void StartTimer()
             }
         }
 
+        /// <summary>
+        /// This will post an event containing the information about patching on CoordinatorService.
+        /// </summary>
+        private void PostWUUpdateEventOnServicePackage()
```

Comment:
>PostWUUpdateEventOnServicePackage [](start = 21, length = 33)

We are no longer posting on ServicePackage. Shouldnt this method name be udpated?

Reply:
will do